wai-app-static/Network/Wai/Application/Static.hs

```diff
@@ -214,10 +214,10 @@ staticApp set req = staticAppPieces set (W.pathInfo req) req
 
 staticAppPieces :: StaticSettings -> [Text] -> W.Application
 staticAppPieces _ _ req sendResponse
-    | W.requestMethod req /= "GET" = sendResponse $ W.responseLBS
+    | notElem (W.requestMethod req) ["GET", "HEAD"] = sendResponse $ W.responseLBS
```

Comment:
Minor note, mostly just to pass it on: using `notElem` will be slightly slower than using an `||` and testing each possibility explicitly, since the latter can avoid any list creation/traversal overhead. It's theoretically possible that this would be optimized away, but I generally don't rely on it. In this case, I'm not convinced the performance difference is worth changing the code.